src/blocks/server.blocks/middleware/middleware.node.js

```diff
@@ -28,7 +30,7 @@ modules.define('middleware', [
                 slashes,
                 redirect,
                 router,
-                htmlCache,
+                config.get('NODE_ENV') !== 'development' ? htmlCache : next,
```

Comment:
why not to add this check into `htmlCache` module and get rid of `middleware__next`?